### PR TITLE
fix: rate expression in flow-time-updater lambda

### LINKED ISSUE #334: Flow Time Updater lambda produces invalid scheduled rate expressions
When creating an `OnScheduleFlow`, the flow time updater produces cron expressions that aren't compatible with the AppFlow API expectations.

According to [the docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appflow-flow-scheduledtriggerproperties.html#aws-properties-appflow-flow-scheduledtriggerproperties-properties), they require rate expressions that look like `rate(5minutes)`.

However, the custom resource lambda is not producing rates that look like this. With an input like `rate(1 day)` (which is what CDK events produce from `Schedule.rate(Duration.days(1))`, the lambda produces an output of `rate(1 days)`. This is close (it pluralized `day` -> `days`), but the space needs to be removed.

I haven't validated if the flow actually runs, but my assumption is that it does not due to this message in the UI:

Screenshot 2024-09-09 at 11 05 38

PS: thank you for this construct - it has been immensely helpful!

### PR DESCRIPTION
**Pull request by blimmer**
Fixes #334

### CODE REVIEW

File: test/core/flows/flow-time-updater/flow-time-updater-function.test.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Duration } from 'aws-cdk-lib';
+import { Schedule } from 'aws-cdk-lib/aws-events';
+import type { CloudFormationCustomResourceEvent } from 'aws-lambda';
+import * as API from '../../../../src/core/flows/flow-time-updater/api';
+import { handler } from '../../../../src/core/flows/flow-time-updater/flow-time-updater.lambda';
+
+describe('flow time updater function', () => {
+  it('converts CDK rate expressions to AppFlow rate expressions', async () => {
```
Comment (blimmer) [inline test/core/flows/flow-time-updater/flow-time-updater-function.test.ts:8]: I added a test for the new behavior. I didn't have the bandwidth to backfill other tests, but this test file could be updated in the future to improve test coverage.

**Review by rpawlaszek — APPROVED** [review]
The changes are clear, the PR is ffocused and does resolve the UX/UI issue preventing editing scheduled flow. The test makes sure that future changes do not break it. All is good.

**Comment by rpawlaszek** [comment]
@blimmer, apart from the test you need to update the snapshots of the onschedule integ tests as the change in the updater effectively changed the name of the asset and thus - the tests failed. Once you've got that we'll be ready to merge.

**Comment by blimmer** [comment]
Fixed!

**Review by rpawlaszek — APPROVED** [review]
All good.

**Pull request merged**